src/index.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './redux-js/index';
-import {store} from "./redux-js/stores/index";
+import {Provider} from 'react-redux';
 import App from './App';
+import './redux-js/index';
+import registerServiceWorker from './registerServiceWorker';
+
 ReactDOM.render(
-     <provider store={store}>
-         <App />
-     </provider>
-         , document.getElementById('root'));
+	<Provider store={window.store}>
```

Comment:
What is the relevance of adding `{window.store}` instead of maintaining `{store}`?

Reply:
By using window.store in the place of `store` makes the store global

Reply:
Okay then. 
LGTM